server/service/integration_mdm_test.go

```diff
@@ -2185,8 +2203,7 @@ func (s *integrationMDMTestSuite) TestMDMAppleUnenroll() {
 	}}, http.StatusNoContent)
 
 	// trigger a sync and verify that there are profiles assigned to the host
-	_, err = s.profileSchedule.Trigger()
-	require.NoError(t, err)
+	s.awaitTriggerProfileSchedule(t)
```

Comment:
unrelated change but this fixes the random failures on this test (succeeds https://github.com/fleetdm/fleet/pull/14954)